src/InfoContainer.js

```diff
@@ -0,0 +1,39 @@
+import info from "./images/toolsimage.png";
+import info2 from "./images/toolsimage2.png";
+import info3 from "./images/toolsimage3.png";
+import info4 from "./images/toolsimage4.png";
+
+function InfoContainer() {
+  return (
+    <div className="InfoContainer">
+      <div className="firsticon">
+        <img src={info} alt="info" srcset="" />
+        <h4> Declarative</h4>
+        <p>React makes it</p>
+        <p> painless to create</p>
+        <p>interactive UIs.</p>
+      </div>
```

Comment:
You shoud create a new component, everytime you see repeated code, and then just call it when you need it.